scripts/cs2/run_compareScenarios2.R

```diff
@@ -88,4 +88,4 @@ run_compareScenarios2 <- function(
   message("Done!\n")
 }
 
-run_compareScenarios2(outputDirs, outFileName, profileName)
```

Comment:
couldn't you just revert any changes to this file as it apparently has no difference added in your PR?

Reply:
done